dbt/adapters/athena/utils.py

```diff
@@ -9,7 +10,17 @@ def clean_sql_comment(comment: str) -> str:
 
 
 def get_catalog_id(catalog: Optional[DataCatalogTypeDef]) -> Optional[str]:
-    return catalog["Parameters"]["catalog-id"] if catalog else None
+    return catalog["Parameters"]["catalog-id"] if catalog and catalog["Type"] == "GLUE" else None
+
+
+class AthenaCatalogType(Enum):
```

Comment:
I think we can create separate module `catalog` and move there all corresponding methods about catalog from here and `impl` module. This will shorten our `impl` module and make adapter a bit more readable. @maiarareinaldo @Jrmyy WDYT can we do it in current PR or do such refactoring in separate one ? `utils` module seems to be not suitable place for such concrete entity as catalog.

Reply:
all right. then I'll close this thread for now